Makefile

```diff
@@ -22,7 +22,7 @@ SECURITY_USER_NAME := $(or $(SECURITY_USER_NAME), aws-broker)
 SECURITY_USER_PASSWORD := $(or $(SECURITY_USER_PASSWORD), aws-broker-pw)
 GSB_COMPATIBILITY_ENABLE_BETA_SERVICES := $(or $(GSB_COMPATIBILITY_ENABLE_BETA_SERVICES), true)
 export GSB_SERVICE_CSB_GOOGLE_POSTGRES_PLANS = [{"name":"small","id":"5b45de36-cb90-11ec-a755-77f8be95a49d","description":"PostgreSQL with default version, shared CPU, minimum 0.6GB ram, 10GB storage","display_name":"small","tier":"db-f1-micro","storage_gb":10},{"name":"medium","id":"a3359fa6-cb90-11ec-bcb6-cb68544eda78","description":"PostgreSQL with default version, shared CPU, minimum 1.7GB ram, 20GB storage","display_name":"medium","tier":"db-g1-small","storage_gb":20},{"name":"large","id":"cd95c5b4-cb90-11ec-a5da-df87b7fb7426","description":"PostgreSQL with default version, minimum 8 cores, minimum 8GB ram, 50GB storage","display_name":"large","tier":"db-custom-8-8192","storage_gb":50}]
-export GSB_SERVICE_CSB_GOOGLE_MYSQL_PLANS = [{"name": "default","id": "eec62c9b-b25e-4e65-bad5-6b74d90274bf","description": "Default MySQL v5.7 10GB storage","display_name": "default","mysql_version": "MYSQL_5_7","storage_gb": 10,"tier": "db-n1-standard-2"}]
+export GSB_SERVICE_CSB_GOOGLE_MYSQL_PLANS = [{"name": "default","id": "eec62c9b-b25e-4e65-bad5-6b74d90274bf","description": "Default MySQL v8.0 10GB storage","display_name": "default","mysql_version": "MYSQL_8_0","storage_gb": 10,"tier": "db-n1-standard-2"}]
```

Comment:
For reasons, we duplicate this in the brokerpak (at least in `.envrc`, `env.go` and probably other places). It's not ideal, but we should update the other locations too so that they remain consistent.

Reply:
Updated all the other locations